src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/DeleghePagoPATest.java

```diff
@@ -220,6 +228,17 @@ public void siSceglieOpzioneAccetta() {
         this.leTueDelegheSection.clickOpzioneAccetta();
     }
 
+    @And("Si inserisce il codice delega nel pop-up OTP")
+    public void inserisceCodiceOTPDelega(){
+        this.leTueDelegheSection.waitPopUpLoad();
+        this.leTueDelegheSection.inserireCodiceDelega(codiceVerifica);
+    }
+
+    @And("Nella Pagina Notifiche persona fisica si clicca su notifiche delegate")
```

Comment:
this step already exist

Reply:
it is new, because in the case of PF locator is different for each delegato

Reply:
pass the name and update atomic

Reply:
per creare notifica con questo mittente, dobbiamo fare un grande refactor su metodo di Gabri. questo modo di creare la notifica e piu veloce e comodo per questo utenza